docs/pages/converting_spectra/fits_header_modify.ipynb

```diff
@@ -0,0 +1,192 @@
+      "\nDuring handling of the above exception, another exception occurred:\n",
+      "\u001b[0;31mUnboundLocalError\u001b[0m                         Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[8], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m \u001b[43mcheck_header\u001b[49m\u001b[43m(\u001b[49m\u001b[43mheader_2144\u001b[49m\u001b[43m)\u001b[49m\n",
+      "File \u001b[0;32m~/miniforge3/envs/template-dev/lib/python3.13/site-packages/astrodb_utils/fits.py:198\u001b[0m, in \u001b[0;36mcheck_header\u001b[0;34m(header, format, ignore_simbad)\u001b[0m\n\u001b[1;32m    195\u001b[0m         missing_keywords\u001b[38;5;241m.\u001b[39mappend((keyword, comment))\n\u001b[1;32m    196\u001b[0m         \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mkeyword\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m : \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mcomment\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)            \n\u001b[0;32m--> 198\u001b[0m coord \u001b[38;5;241m=\u001b[39m \u001b[43mmake_skycoord\u001b[49m\u001b[43m(\u001b[49m\u001b[43mheader\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    199\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m coord \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m    200\u001b[0m     \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mCoordinates are required\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
+      "File \u001b[0;32m~/miniforge3/envs/template-dev/lib/python3.13/site-packages/astrodb_utils/fits.py:318\u001b[0m, in \u001b[0;36mmake_skycoord\u001b[0;34m(header)\u001b[0m\n\u001b[1;32m    316\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m coord\n\u001b[1;32m    317\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m--> 318\u001b[0m     \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mcoordinates (\u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[43mra\u001b[49m\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m,\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mdec\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m) could not be converted to Skycoord object: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00me\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    319\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m\n",
+      "\u001b[0;31mUnboundLocalError\u001b[0m: cannot access local variable 'ra' where it is not associated with a value"
+     ]
+    }
+   ],
+   "source": [
+    "check_header(header_2144)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## RA_Targ and Dec_targ are required so let's add them using the RA/Dec in the header"
```

Comment:
do the last two cells have any output?

Reply:
Yes, re-ran the noteebook so it displays.